tox.ini

```diff
@@ -135,6 +135,14 @@ commands = {envpython} -m pytest \
             -m "not hypothesis_slow" \
             {posargs:--cov=cloudinit --cov-branch tests/unittests}
 
+# experimental
+[testenv:py3-fast]
+deps =
+    -r{toxinidir}/test-requirements.txt
+    pytest-xdist
+commands = {envpython} -m pytest -n auto -m "not hypothesis_slow" -m "not serial"\
```

Comment:
Note that `--dist=loadscope` would probably result in similar times without requiring skipping any tests.

https://stackoverflow.com/a/56475200

Reply:
This PR originally used loadscope, but when I switched away from it (and simultaneously removed the cov/duration tracking bits) it ended up about 20s faster. I didn't bother finding the source of the speedup so I don't know whether that's caused by loadscope or cov or duration tracking or something else entirely. Alternatively we could use loadgroup and mark them as groups, but when I did that I saw errors I didn't feel like debugging so I reverted back to this. Laziness prevailed here I guess.